app/models/healthcheck.rb

```diff
@@ -41,7 +41,7 @@ def test_dfe_signin_api
     return true unless Schools::DFESignInAPI::Organisations.enabled?
 
     Schools::DFESignInAPI::Organisations.new(SecureRandom.uuid).uuids
-  rescue RuntimeError
+  rescue RuntimeError, Rack::Timeout::RequestTimeoutException
```

Comment:
Worth a quick test? 🙂

Reply:
you might actually find that calling this is better https://services.signin.education.gov.uk/healthcheck

Reply:
@ethax-ross uh, for some reason I thought I had created tests :sweat_smile: . I added them now.

@mattonfoot that's a good one, thanks. I believe we need to keep the current check (which is an integration check that can also indicate issues on our side) and also use the https://services.signin.education.gov.uk/healthcheck as an additional check. However, I would rather do that in another PR.